pkg/services/accesscontrol/ossaccesscontrol/ossaccesscontrol.go

```diff
@@ -55,9 +54,17 @@ func (ac *OSSAccessControlService) getUsageMetrics() interface{} {
 	return 1
 }
 
-// Evaluate evaluates access to the given resource
-func (ac *OSSAccessControlService) Evaluate(ctx context.Context, user *models.SignedInUser, permission string, scope ...string) (bool, error) {
-	return evaluator.Evaluate(ctx, ac, user, permission, scope...)
+// Evaluate evaluates access to the given resources
+func (ac *OSSAccessControlService) Evaluate(ctx context.Context, user *models.SignedInUser, evaluator accesscontrol.Evaluator) (bool, error) {
+	timer := prometheus.NewTimer(metrics.MAccessEvaluationsSummary)
+	defer timer.ObserveDuration()
+	metrics.MAccessEvaluationCount.Inc()
+
+	permissions, err := ac.GetUserPermissions(ctx, user)
+	if err != nil {
+		return false, err
+	}
+	return evaluator.Evaluate(accesscontrol.GroupScopesByAction(permissions))
```

Comment:
Hm, we should look at how we can make this operation cheaper. With a lot of permissions, this will be costly.

Reply:
So in oss this wont result in a db query.  but it will in enterprise. We could add a function to just fetch permissions for a specific action? But then we have to solve that for caching as well

Reply:
(to clarify: not for this PR)